_docs/howto/deploy/change-gcp-deployment.md

```diff
@@ -27,97 +27,86 @@ will have a timestamp suffix, e.g. `deploy-forseti-20171001000000.yaml`.
+     
+     ```bash
+     $ git checkout v1.1.3
+     ```
 
 3. You will use this release version in your `deploy-forseti.yaml` file as follows:
 
-   "I want to deploy `master` branch":
-   
-   ```yaml
-   branch-name: "master"
-   # release-version: ...
-   ```
-   
-   "I want to deploy Release 1.1.3":
-   
-   ```yaml
-   # branch-name: ...
-   release-version: "1.1.3"
-   ```
-
-## Check for differences
-
-1. Check `deploy-forseti.yaml.sample` to see if there are any new properties 
-   that you need to copy over to your previously generated 
-   `deploy-forseti-<TIMESTAMP>.yaml`. You can use `git diff` to compare what 
-   changed. For example, to see the diff between the latest (HEAD) and one revision ago:
-
-   ```bash
-   $ git diff origin/master..HEAD~1 -- deploy-forseti.yaml.sample
-   ```
+    "I want to deploy `master` branch":
+     
+     ```yaml
+     branch-name: "master"
+     # release-version: ...
+     ```
+     
+     "I want to deploy Release 1.1.3":
+     
+     ```yaml
+     # branch-name: ...
+     release-version: "1.1.3"
+     ```
+
+## Check for differences and make changes
+
+There are likely changes to the deployment template between versions. It's important to check for
+and include values for any new deployment variables.
+
+1. Check [`deploy-forseti.yaml.in`](https://github.com/GoogleCloudPlatform/forseti-security/blob/master/deployment-templates/deploy-forseti.yaml.in)
+  to see if there are any new properties that you need to copy over to your previously generated 
+  `deploy-forseti-<TIMESTAMP>.yaml`. You can use `git diff` to compare what 
+  changed. For example, to see the diff between the latest (HEAD) and one revision ago:
+
+     ```bash
+     $ git diff origin/master..HEAD~1 -- deploy-forseti.yaml.in
+     ```
 
 1. Edit `deploy-forseti-<TIMESTAMP>.yaml` and update the values of the new properties.
 
-For example, from v1.1.7 to v1.1.10, the following Compute Engine instance 
-properties have been changed and/or added.
+    For example, from v1.1.7 to v1.1.10, the following Compute Engine instance 
+    properties have been changed and/or added.
 
-   ```yaml
-   region: $(ref.cloudsql-instance.region)
+     ```yaml
+     region: $(ref.cloudsql-instance.region)
 
-   network-host-project-id: YOUR_PROJECT_ID
-   vpc-name: default 
-   subnetwork-name: default 
-   ```
+     network-host-project-id: YOUR_PROJECT_ID
+     vpc-name: default 
+     subnetwork-name: default 
+     ```
 
-To upgrade, copy these new properties to your generated 
-`deploy-forseti-<TIMESTAMP>.yaml`. Then, update the placeholders to the values 
-you want to use. For example, the id of the project that Forseti is 
-running in, "default", "default".
+    To upgrade, copy these new properties to your generated 
+    `deploy-forseti-<TIMESTAMP>.yaml`. Then, update the placeholders to the values 
+    you want to use.
 
 1. Inspect `deploy-forseti-<TIMESTAMP>.yaml` and verify if your `branch-name` 
```

Comment:
Just to clarify, I believe we can either use property `branch-name` or `release-version` in the template, but `branch-name` is something like "master" and shouldn't have any reference to a specific version?

Reply:
Thanks, I've clarified it.